### ISSUE TITLE
Can't Join on Win10 Edition: Cannot write non-existing field "__encapsulatedPacket"

### ISSUE THREAD

**Issue by Catboy96** [pmmp/PocketMine-MP#2542]
### Issue description
When joining on Windows 10 Edition, PMMP got an error like this:
```
Could not execute completion of asynchronous task VerifyLoginTask: Cannot write non-existing field "__encapsulatedPacket"  
Error: "Cannot write non-existing field "__encapsulatedPacket"" (EXCEPTION) in "src/pocketmine/network/mcpe/protocol/DataPacket" at line 143
```
Windows 10 Edition can detect the PMMP server's information, like server name, players and ping status. But it cannot join.  
I've tested on multiple Virtual Private Servers like DigitalOcean and Vultr and got the same issue.

- Expected result: Successully join PMMP server on Windows 10 Edition.
- Actual result: Could not join and got a server-side error.

### Steps to reproduce the issue
1. Create a non-root account on Debian 9.
2. Do a clean install of PMMP on Debian 9 amd64 using 'https://get.pmmp.io' method.
3. Start the server using `./start.sh`
4. Open Windows 10 Edition, enter server details and join.

### OS and versions
* PocketMine-MP: 3.3.3 for Minecraft: Bedrock Edition v1.7.0 (protocol version 291)
* PHP: 7.2.11 (cli)
* Server OS: Debian 9 64-bit
* Game version: Win10 v1.7.1

### Server's hardware information
CPU: Intel(R) Xeon(R) CPU E5-26xx v3 with 2 Cores
RAM: 2GB

### Plugins
There's NO plugins installed.

### Crashdump, backtrace or other files
`Crashdump`, `pocketmine.yml` and `server.properties` are followed:
https://gist.github.com/Catboy96/ff346a84e6451d56d79b08ed4065a85a

**Comment by gouchaoer** [comment]
the same problem
phar version:  PocketMine-MP 3.3.3

**Comment by CupidonSauce173** [comment]
same issue.

**Comment by Nathaniel200412** [comment]
Same

**Comment by thnuder** [comment]
Same problem after upgrade to version 3.3.3 on IOS / Android
reverting back to version 3.3.2 works

**Comment by Jacobboogiebear** [comment]
Same here

**Comment by CortexPE** [comment]
Can confirm

**Comment by dktapps** [comment]
guys, if all you're going to do is write "same", you might as well not bother. It's just a bunch of useless comments that a reader has to scroll through. Reactions are sufficient.

**Comment by dktapps** [comment]
Fixed by 1a6db1c7cede142c86c9df41686180369f6cfa8c. Please install the 3.3.4 release.
